CHANGELOG.md

```diff
@@ -1,25 +1,25 @@
 ## 2.1.0 (March 25, 2019)
 
-ARCHITECTURAL:
+BREAKING CHANGES:
```

Comment:
```suggestion
NOTES:
```

I think these are rather NOTES than BREAKING CHANGES. User experience is not broken and as per Hashicorp `This should only be present in major version upgrades.`

Reply:
ok. :)